bulk-install-app/debug.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from config import DEBUG
+
+
+def save(folder, name, response):
+    if DEBUG:
```

Comment:
it seems like we call save a lot. Is it just for debugging purposes? I guess I was just confused by the name save instead of something like save_debug_info()

Reply:
Yeah, it's just for debugging. Will rename in a follow-up!